sbtc-core/src/lib.rs

```diff
@@ -37,6 +37,9 @@ pub enum SBTCError {
     /// Stacks error
     #[error("Stacks error: {0}")]
     StacksError(#[from] StacksError),
+    /// Invalid Config File
+    #[error("Invalid config file")]
+    InvalidConfig(String),
```

Comment:
Similar to @netrome I'm also wondering if signer should have it's own error enum and result type. It feels like signer is different enough and has a domain separate enough, but I'm still not sure. What do you think @jferrant?